src/script/assets/AssetCrypto.js

```diff
@@ -17,21 +17,18 @@
  *
  */
 
-window.z = window.z || {};
-window.z.assets = z.assets || {};
-
-z.assets.AssetCrypto = (() => {
+export default class AssetCrypto {
```

Comment:
I believe this will not work. Previously we exported an object, not a class (which means we don't do `new` on the other side).
I think an object is the right abstraction for this. 

side note: we are moving away from default exports to named ones